script.js

```diff
@@ -1,4 +1,268 @@
+var currentGameMode = "Start";
+var numofPlayers = 0;
+var numofDice = 0;
+var currentPlayerDiceResults = [];
+
+var numofRounds = 0;
+var numofTurns = 0;
+var totalPlayers = [];
+var remainingPlayers = [];
+var currentRoundPlayersWon = [];
+var currentPlayer = 0;
+var indexCurrentPlayer = 0;
+
+var everyPlayerCurrentScore = [];
+
+var rollDice = function () {
+  var diceOutcome = Math.floor(Math.random() * 6) + 1;
+  return diceOutcome;
+};
+
+var selectPlayer = function () {
+  var index = Math.floor(Math.random() * (numofPlayers - numofTurns));
+  var playerSelected = remainingPlayers[index];
+
+  remainingPlayers.splice(remainingPlayers.indexOf(playerSelected), 1);
+
+  // for (var counter = 0; counter < index; counter += 1) {
+  //   remainingPlayers.push(remainingPlayers.shift());
+  // }
+
+  // remainingPlayers.shift();
+
+  return playerSelected;
+};
```

Comment:
Impressive use of splice and Math methods to determine the player and the players remaining!